app/decorators/physical_server_decorator.rb

```diff
@@ -0,0 +1,14 @@
+class PhysicalServerDecorator < MiqDecorator
+  def img_for_physical_vendor
+    "svg/vendor-#{label_for_vendor.downcase}.svg"
+  end
+
+  def img_for_health_state
```

Comment:
Just following the MIQ pattern.(e. g.: `img_for_compliance`, `img_for_vendor`, `img_for_host_vendor` and so on...)